AUTHORS.md

```diff
@@ -7,14 +7,15 @@ but over time many contributors helped make it much better, including (among oth
 * [Adithya Obilisetty](mailto:adi.obilisetty@gmail.com) [@adithyaov](https://github.com/adithyaov)
 * [Alexandre Moine](mailto:alexandre@moine.me) [@nobrakal](https://github.com/nobrakal)
 * [Armando Santos](mailto:armandoifsantos@gmail.com) [@bolt12](https://github.com/bolt12)
+* Patrick Hilhorst [@Synthetica9](https://github.com/synthetica9)
 * [Piotr Gawryś](mailto:pgawrys2@gmail.com) [@Avasil](https://github.com/Avasil)
 
-If you are not on this list, it's not because your contributions are not appreciated, but 
+If you are not on this list, it's not because your contributions are not appreciated, but
 because I didn't want to add your name and contact details without your consent. Please fix this
 by sending a PR, keeping the list alphabetical.
 
 Also see the autogenerated yet still possibly incomplete
 [list of contributors](https://github.com/snowleopard/alga/graphs/contributors).
 
-Thank you all for your help!  
```

Comment:
These spaces are important for rendering, please undo whitespace changes in this file.